gf-webapp/src/component/pageBorder/header/languagesList/LanguagesList.tsx

```diff
@@ -0,0 +1,56 @@
+import {useState, useEffect} from "react";
+import {useDictionary} from "src/logic/DictionaryContext/useDictionary";
+import {useLanguage} from "src/service/Language/useLanguage";
+import {useScrollPosition} from "src/utils/domEventsUtils/useScrollPosition";
+import {loadDictionary} from "src/service/Language/LoadDictionary";
+import {Language, languageList} from "src/model/Language";
+import styles from "src/component/pageBorder/header/languagesList/LanguagesList.module.scss";
+
+export const LanguagesList = () => {
+  const {dictionary, setDictionary} = useDictionary();
+  const [langOpen, setLangOpen] = useState(false);
+  const {lang, setCurrentLang} = useLanguage();
+
+  const langHoverHandler = () => {
+    setLangOpen(prev => !prev);
+  };
```

Comment:
Here I like the sequence of variable declarations. You can take this as an example and use it in other files.

You don't use a dictionary from useDictionary(), you should remove it

Reply:
Done